app/renderer/ui/components/main/sections/wallet/style.scss

```diff
@@ -1,7 +1,6 @@
 :local {
   .layout {
     display: grid;
-    grid-gap: 2em;
```

Comment:
After removing the grid gap, then the columns are too closed to each other.

Reply:
Done